docs/content/preview/explore/ycql-language/keyspaces-tables.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Keyspaces and tables
+linkTitle: Keyspaces and tables
+description: Keyspaces and tables in YCQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  preview:
+    identifier: explore-ycql-language-features-keyspaces-tables
+    parent: explore-ycql-language
+    weight: 60
+type: docs
+---
+
+This section explores the topics of Keyspaces and Tables in YCQL using the command line shell
+`ycqlsh`.
```

Comment:
line 15 can be deleted as I have already included `ycqlsh` in the previous suggested change.